src/Bigtable/Connection/ConnectionInterface.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/*
+ * Copyright 2018, Google LLC All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Bigtable;
+
+/**
+* Represents a connection to Cloud Bigtable.
+*/
+interface ConnectionInterface
+{
+
+    /**
+    * @param array $args
+    */
+    public function CreateInstance(array $args);
```

Comment:
Method names must conform to [PSR-2](https://www.php-fig.org/psr/psr-2/), which mandates camelCase names. i.e. `createInstance`. Heading in the right direction though! :)

Reply:
It may worth running a tool like [PHP Code Sniffer](https://github.com/squizlabs/PHP_CodeSniffer) for all your classes in the PR. :)

Reply:
Most definitely!

We include a script to check code style with all the proper configuration for the library:

```sh
$ dev/sh/style
```